operators/kogito-operator/1.28.0/manifests/kogito-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,1024 @@
+                - /manager
+                env:
+                - name: BUILDER_IMAGE
+                  value: kogito-s2i-builder
+                - name: RUNTIME_IMAGE
+                  value: kogito-runtime-jvm
+                - name: RUNTIME_NATIVE_IMAGE
+                  value: kogito-runtime-native
+                - name: IMAGE_REGISTRY
+                  value: quay.io/kiegroup
+                image: quay.io/kiegroup/kogito-operator:2.0.0-snapshot
+                livenessProbe:
+                  httpGet:
+                    path: /healthz
+                    port: 8081
+                  initialDelaySeconds: 15
+                  periodSeconds: 20
+                name: manager
+                readinessProbe:
+                  httpGet:
+                    path: /readyz
+                    port: 8081
+                  initialDelaySeconds: 5
+                  periodSeconds: 10
+                resources:
+                  limits:
+                    cpu: 200m
+                    memory: 200Mi
+                  requests:
+                    cpu: 100m
+                    memory: 20Mi
+                securityContext:
+                  allowPrivilegeEscalation: false
+              securityContext:
+                runAsNonRoot: true
+              serviceAccountName: kogito-operator-controller-manager
+              terminationGracePeriodSeconds: 10
+      permissions:
+      - rules:
+        - apiGroups:
+          - ""
+          resources:
+          - configmaps
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - coordination.k8s.io
+          resources:
+          - leases
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - ""
+          resources:
+          - events
+          verbs:
+          - create
+          - patch
+        serviceAccountName: kogito-operator-controller-manager
+    strategy: deployment
+  installModes:
+  - supported: false
+    type: OwnNamespace
+  - supported: false
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: true
+    type: AllNamespaces
+  keywords:
+  - cloud
+  - kogito
+  - quarkus
+  - springboot
+  - operator
+  - native
+  - automation
+  labels:
+    alm-owner-kogito: kogito-operator
+    operated-by: kogito-operator.2.0.0-snapshot
+  links:
+  - name: Product Page
+    url: https://kogito.kie.org/
+  - name: Documentation
+    url: https://github.com/kiegroup/kogito-operator/blob/main/README.md
+  - name: Source
+    url: https://github.com/kiegroup/kogito-operator/
+  maintainers:
+  - email: bsig-cloud@redhat.com
+    name: Red Hat
+  maturity: beta
+  provider:
+    name: Red Hat
+  replaces: kogito-operator.v1.9.0
+  version: 2.0.0-snapshot
```

Comment:
@spolti version should be same as directory name (1.28.0)

Reply:
yeah, it is wrong, usually our CI does the update version, not sure what happened, will take a look on Monday, thanks for the heads up.

Reply:
updated.